guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -197,6 +258,20 @@ If `eth0` was renamed to `em0`, call the installer to use the new interface name
 # leapp upgrade
 ----
 
+ifdef::satellite[]
++
+. If you run {Project} in a disconnected environment, add the `--no-rhsm` and `--enablerepo` parameters:
```

Comment:
This should be additional information to the previous `leapp upgrade` step, not an own step.